packages/backend-api/src/api/services/updateNotifications.ts

```diff
@@ -122,6 +123,28 @@ async function getSystemData() {
   } as IMessage;
 }
 
+// Convert IDs to strings, and assignedModerators to arrays of strings.
+function serialiseObject(o: Sequelize.Instance<any>, fields: Array<string>): {[key: string]: {} | string | number} {
```

Comment:
cool, are you sure you can change `any` in the type param of `Sequelize.Instance` to something more specific? at the very least maybe `{}`, although looking at the code below, it seems that it maybe is at least `{ id:string }` ?

Reply:
It needs to be one of the IXXXAttributes, where XXX is Article, Comment, Tag etc.  I can enumerate the possibilities if you like, though I don't think it actually buys us anything.  This interface is generic.  Though you are right, {} is probably better than any.

We get the ID field directly from the Sequelize.Instance type (I believe).  I don't think the IXXXAttributes type actually needs it, and a few of our types don't contain it.  Having said that, I'm still struggling with the types for Sequelize objects.  

(As I may have mentioned, I'm planning on doing a rework of the types when I upgrade sequelize.  In fact, this is mandatory as the current code doesn't compile with upgraded Sequelize types, even when the JS interface hasn't changed.  I don't know what is going on there, but it can't be good.)

Reply:
ok, then I guess having this as `{}` is good enough for now :)